frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -44,8 +44,6 @@ export const UpsellCTALink = styled(ExternalLink)`
   font-size: 0.75rem;
   padding: 0.25rem 0.5rem;
   border-radius: 2rem;
-  margin-inline: 1rem;
-  margin-bottom: 1.5rem;
```

Comment:
It's best if we let the component adapt to different surroundings and contexts, and keep only the visual styles in here, imo.